github/client.go

```diff
@@ -152,6 +152,67 @@ func (c *Client) ReadReleaseByTag(ctx context.Context, tkn string, repoURL url.U
 	return model.ToSvcGithubRelease(rls)
 }
 
+// TODO once svcmodel.Release has git tag field, no gitTag parameter will be needed
+func (c *Client) UpdateRelease(ctx context.Context, tkn string, repo svcmodel.GithubRepo, rls svcmodel.Release, gitTag string) error {
+	// Release can be only updated by release ID
+	// Therefore release ID is fetched by tag name first
+	id, err := c.getReleaseID(ctx, tkn, repo, gitTag)
+	if err != nil {
+		return fmt.Errorf("failed to get release ID: %w", err)
+	}
+
+	_, _, err = c.getGithubClient(tkn).Repositories.EditRelease(
+		ctx,
+		repo.OwnerSlug,
+		repo.RepoSlug,
+		id,
+		&github.RepositoryRelease{
+			Name: &rls.ReleaseTitle,
+			Body: &rls.ReleaseNotes,
+		},
+	)
+	if err != nil {
+		return fmt.Errorf("failed to update release: %w", err)
+	}
+
+	return nil
+}
+
+func (c *Client) DeleteRelease(ctx context.Context, tkn string, repo svcmodel.GithubRepo, tagName string) error {
```

Comment:
Already passing `svcmodel.GithubRepo` here instead of url. Will be updated for all github client functions once https://github.com/jan-zabloudil/release-manager/pull/117 is merged